### DISCUSSION TITLE
Task 3 run the card trick - error de ejecución

### DISCUSSION THREAD

**Discussion by carlosdac55** [rramosp/ai4eng.v1#43] [02 - PYTHON]
![error run](https://user-images.githubusercontent.com/85840609/124330530-356c7180-db53-11eb-96fe-93ac23a8fac6.JPG)

Buenas tardes, he estado varios días desarrollando esta tarea y aunque al ejecutar el código me salían errores, el que me esta saliendo desde hoy  nunca antes lo había obtenido, incluso he intentado con código anterior donde este se ejecutaba y obtenía un resultado diferente del esperado, hoy en vez de obtener el error esperado me sale el que se ve en la imagen, incluso nombrando la variable r=6 directamente obtengo el mismo error, en resumen cualquier código que yo escriba me produce ese error. Solicito ayuda por favor para entender porque se origina este y como solucionarlo.

Muchas gracias

**Comment by Felipe-RA** [comment]
Greetings carlos

Este error indica que se intentó enviar la función "make_heaps" pero no se encontró en el diccionario de funciones.

Asegurate que la función esta definida y es accesible.

Si tienes más problemas con esto comentamelo y buscamos una solución.

Hasta pronto 👍
